src/gui/folder.cpp

```diff
@@ -903,20 +901,18 @@ void Folder::slotItemCompleted(const SyncFileItemPtr &item)
 
     // add new directories or remove gone away dirs to the watcher
     if (_folderWatcher && item->isDirectory()) {
-    switch (item->_instruction) {
-        case CSYNC_INSTRUCTION_NEW:
+	switch (item->_instruction) {
+	    case CSYNC_INSTRUCTION_NEW:
                 _folderWatcher->addPath(path() + item->_file);
-        break;
-        case CSYNC_INSTRUCTION_REMOVE:
+	    break;
+	    case CSYNC_INSTRUCTION_REMOVE:
                 _folderWatcher->removePath(path() + item->_file);
-        break;
-        case CSYNC_INSTRUCTION_RENAME:
+	    break;
+	    case CSYNC_INSTRUCTION_RENAME:
                 _folderWatcher->removePath(path() + item->_file);
                 _folderWatcher->addPath(path() + item->destination());
-        break;
-        default:
-        break;
```

Comment:
IMHO an empty default case is better than none (IIRC missing default should also generate a compiler warning?)